Source/WebKit/NetworkProcess/PrivateClickMeasurement/PrivateClickMeasurementManagerInterface.h

```diff
@@ -44,7 +44,7 @@ class PrivateClickMeasurementManager;
 
 namespace PCM {
 
-class ManagerInterface {
+class ManagerInterface : public RefCounted<ManagerInterface> {
```

Comment:
I'm surprised @geoffreygaren liked this. In earlier PRs, I remember him saying that interfaces should provide a virtual ref() / deref() instead and let subclasses decide how to do their refcounting.